src/components/Avatar/README.md

```diff
@@ -188,14 +188,15 @@ LANDING_BLOCK-->
 
 ### Image-specific
 
-| Name           | Description                               |        Type        |   Default   |
-| :------------- | :---------------------------------------- | :----------------: | :---------: |
-| imgUrl         | `img` `src` HTML attribute                |      `string`      |             |
-| fallbackImgUrl | Fallback image shown if an error occurred |      `string`      |             |
-| sizes          | `img` `sizes` HTML attribute              |      `string`      |             |
-| srcSet         | `img` `srcSet` HTML attribute             |      `string`      |             |
-| alt            | `img` `alt` HTML attribute                |      `string`      | props.title |
-| loading        | `img` `loading` HTML attribute            | `'eager'` `'lazy'` |             |
+| Name            | Description                               |        Type        |   Default   |
+| :-------------- | :---------------------------------------- | :----------------: | :---------: |
+| imgUrl          | `img` `src` HTML attribute                |      `string`      |             |
+| fallbackImgUrl  | Fallback image shown if an error occurred |      `string`      |             |
+| sizes           | `img` `sizes` HTML attribute              |      `string`      |             |
+| srcSet          | `img` `srcSet` HTML attribute             |      `string`      |             |
+| alt             | `img` `alt` HTML attribute                |      `string`      | props.title |
+| loading         | `img` `loading` HTML attribute            | `'eager'` `'lazy'` |             |
+| withImageBorder | Add default border for the image          |     `boolean`      |             |
```

Comment:
This seems conflicting with borderColor props

It is completely separate props, that not relate to each other, but produce very similar effect